### DISCUSSION TITLE
ESP32 플래싱에서 에러가뜨면서 플래싱이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by Stella-repo** [luftaquila/monolith#17] [Q&A] [answered]
안녕하세요 창원대학교 SKID 입니다.
데이터로거 제작 후 ESP32만 먼저 플래싱을 해보려고 했지만 다음과 같은 에러를 내면서 플래싱이 되지않습니다.
ESP32를 플래싱할때 10uF 커패시터를 연결하여 플래싱도 해보았고 EN버튼을 누른상태에서 플래싱도 해보았으나 똑같은 메세지가 뜨며 플래싱되지 않았습니다.

TMA-1 설정 프로그램에서 설정셋팅은 다음과 같습니다.
![image](https://github.com/luftaquila/monolith/assets/93361308/f4b26fe9-2c8a-4600-b8b9-55257255fac3)
여기서 COM포트를 수동으로 설정해줬음에도(COM3에 장치가 연결되어있어 COM입력란에 "3"과 "COM3"둘다 시도해봤습니다.) 똑같은 오류를 내며 플래싱되지 않았습니다.

콘솔창에 나타난 메세지를 함께 올려드리겠습니다.
감사합니다.
[240222_005000 TMA-1 설정프로그램 ESP32 플래싱 오류.txt](https://github.com/luftaquila/monolith/files/14361848/240222_005000.TMA-1.ESP32.txt)

**Comment by Stella-repo** [comment]
telemetry.ino을 이용해서 아두이노IDE를 통해 직접 플래싱했을때 정상적으로 플래싱되는것을 확인했습니다.
다만 TMA-1설정프로그램을 통한 플래싱은 여전히 똑같은 오류가 발생하기때문에 한번 검토해주시면 좋을것같습니다.
감사합니다.

**Reply by luftaquila** [reply, marked as answer]
arduino-cli 쪽의 오류 같네요. 이 프로젝트에서 수정할 수 있는 건 없는 것 같습니다.

업로드가 잘 안 된다면 해결하신 방법처럼 직접 `telemetry.ino` 코드의 몇 가지 설정을 소스코드에 입력하시고 직접 아두이노 IDE로 업로드해도 무방합니다.

개인적으로 다른 노트북에서는 설정 도구 사용해서도 정상적으로 진행된다는 연락을 주셨으니 close 하겠습니다.
